conda_libmamba_solver/shards_subset.py

```diff
@@ -127,22 +168,22 @@ def outgoing(self, node: Node):
         for n in self.neighbors(node):
             yield n, 1
 
-    def shortest(self, start_packages):
+    def shortest(self, root_packages):
```

Comment:
```suggestion
    def shortest_dijkstra(self, root_packages):
```

I am not sure if you want to go there, but I thought if we're adding the ability to have multiple algorithms, might as well use the name?